### DISCUSSION TITLE
Is working on linux?

### DISCUSSION THREAD

**Discussion by feraxhp** [Feel-ix-343/markdown-oxide#146] [Q&A] [answered]
I was trying to install it on Linux. But I don't get any rendering, and I don't know if I am doing it wrong. If someone can help me, would be very nice.

**Comment by Feel-ix-343** [comment]
Hello!

I use this on linux everyday!

Which editor are you using?

**Reply by feraxhp** [reply]
Hi! Thanks for your answer, I am using zed. Is there some additional configuration in order to run it?

**Reply by Feel-ix-343** [reply, marked as answer]
It should work by just installing the zed extension.

Did you try this?

If not, do you have markdown-oxide installed to path?

Also note that markdown-oxide on Zed is a WIP. Zed doesn't implement quite a bit of the language server protocol that markdown-oxide uses, so it doesn't work perfectly.

**Reply by feraxhp** [reply]
So its probably, that the render doesn't work, right?

**Reply by Feel-ix-343** [reply]
hmm I am not quite understanding. 

Could you elaborate on this? or a quick video?

**Reply by feraxhp** [reply]
yes, sorry i was confuse, i thought that markdown-oxide actually renders a preview in real time like it works on obsidian. because that its something i have been searching for a long time.

But now i understand this is not the case. so in consequence i wonder if you could add it in the future, or is something that is not ever planed.

**Reply by Feel-ix-343** [reply]
Hey!

I don't think this would be in the scope of markdown-oxide, however zed provides real-time markdown rendering. 

Try out the `markdown: open preview to the side` command!
